tests/pulse/test_rydberg_hamiltonian.py

```diff
@@ -82,7 +82,10 @@ def test_add_rydberg_hamiltonian(self):
             for op1, op2 in zip(sum_rm.ops, [qml.PauliX(4), qml.PauliZ(8), qml.PauliY(8)])
         )
         assert qml.math.allequal(sum_rm.register, atom_coordinates)
-        assert sum_rm.pulses == [RydbergPulse(1, 2, 3, [4, 8]), RydbergPulse(5, 6, 7, 8)]
+        assert sum_rm.pulses == [
+            RydbergPulse(1, 2, 3, [4, 8]),
+            RydbergPulse(5, 6, 7, 8),
+        ]
```

Comment:
Is there an actual change here? :thinking:

Reply:
Nope, I use the precommit hook in the top-level directory of PL, and that reformats files before I push. I don't know why this change happened since I'd imagine that the constraints that black uses in the Github action are the same the ones the precommit hook uses.

Reply:
maybe a different version of black?

Reply:
That could be it, but I'd imagine there would be a lot more failing black CI checks in that case as well 🤔